components/sst_importer/src/sst_importer.rs

```diff
@@ -308,61 +314,58 @@ impl SstImporter {
         Ok(())
     }
 
-    pub fn shrink_by_tick(&self) {
-        if self.import_support_download() {
-            self.shrink_space_by_tick();
-        } else {
-            self.shrink_cache_by_tick();
-        }
-    }
-
-    pub fn shrink_cache_by_tick(&self) -> u64 {
-        let mut shrink_size: usize = 0;
-        let mut retain_size: usize = 0;
-
-        self.file_locks.retain(|_, (buff, refcnt, start)| {
-            let need_retain =
-                *refcnt > 0_u64 || start.saturating_elapsed() < Duration::from_secs(60);
-            if need_retain {
-                retain_size += buff.len();
-            } else {
-                shrink_size += buff.len();
+    pub fn shrink_by_tick(&self) -> usize {
+        let mut shrink_buff_size: usize = 0;
+        let mut retain_buff_size: usize = 0;
+        let mut shrink_files: Vec<PathBuf> = Vec::new();
+        let mut retain_file_count = 0_usize;
+
+        self.file_locks.retain(|_, (buff, start)| {
+            let mut need_retain = true;
+            match buff {
+                CacheKvFile::Mem(buff) => {
+                    let buflen = buff.len();
+                    // The term of recycle memeory is 60s.
+                    if Arc::strong_count(&buff) == 1
```

Comment:
We can extract the `match` into a method like `CacheKvFile::ref_count()`

Reply:
Done